### DISCUSSION TITLE
Everything works but I can't see anything in VR

### DISCUSSION THREAD

**Discussion by devDariush** [alvr-org/ALVR#1925] [Q&A]
I'm using ALVR on a Paperspace cloud PC with Tailscale on my Pico 4. Everything works well. My headset gets detected and I can start VRChat. All my movements get tracked and I can see everything on the 2D screen but not in VR for some reason. I can only see a blueish-green color on my headset.

**Comment by jrtokarz** [comment]
I have the same issue with a Quest 2 and Linux.   Everything looks good in SteamVR and ALVR (lots of green :-) ), tracking is working fine but nothing is rendered in the headset.

**Comment by FilipoDeveloperMan** [comment]
There's already a working solution which fixed my issue. I'm on arch and using phonvr. Just put ~/.local/share/Steam/steamapps/common/SteamVR/bin/vrmonitor.sh %command% in launch options for steamvr. Should fix it.
